presto-raptor/src/main/java/com/facebook/presto/raptor/storage/ShardEjector.java

```diff
@@ -215,8 +215,8 @@ void process()
         // only include nodes that are below threshold
         nodes = new HashMap<>(filterValues(nodes, size -> size <= averageSize));
 
-        // get non-bucketed node shards by size, largest to smallest
-        List<ShardMetadata> shards = shardManager.getNodeShards(currentNode).stream()
+        // get non-bucketed node shards(only) by size, largest to smallest
```

Comment:
space after shards

Reply:
there is space after this shards actually